resources/views/auth/form.blade.php

```diff
@@ -11,6 +11,11 @@
     {!! Form::text('email', null, ['placeholder' => 'you@example.com']) !!}
 @endif
 
+@if(should_collect_international_phone())
+    {!! Form::label('phone', 'Cell Number (optional)') !!}
+    {!! Form::text('phone', null, ['placeholder' => '555-555-5555']) !!}
```

Comment:
1. I kept it separate so we can add the `(optional)` to the label.
2. Oh boyyyyyyy. I feel like that could get messy to maintain. Lemme make an issue & we can look into how easy translation in Laravel is...